tests/dataDriven_test.js

```diff
@@ -0,0 +1,27 @@
+let data = new DataTable(["url", "statusCode"]);
+data.add([
+  "/us/obituaries/chicagotribune/name/virginia-gruchalski-obituary?pid=196167379",
+  "200",
+]);
+data.add([
+  "/us/obituaries/chicagotribune/name/virginia-gruchalski-obituary?pid=098734232",
+  "404",
+]);
+
+var assert = require("assert");
+Feature("Data Driven");
```

Comment:
I didn't think we used `var` anymore, just `const` and sometimes `let`? Gilbert, can you confirm that this is allowed in the Onboarding Tasks?

Reply:
this has been updated with the latest commit

Reply:
yes, for global variables use `const` and local variables use `let` or `const` depending on the situation.